### ISSUE TITLE
Cannot use `DefaultFlag`

### ISSUE THREAD

**Issue by matthewelwell** [Flagsmith/flagsmith-nodejs-client#115]
Both of the following snippets will log out undefined and attempting to instantiate a `DefaultFlag` object will throw an error, meaning that it's not possible to use the `DefaultFlag` constructor directly when defining default flag handler logic. 

```javascript
import { DefaultFlag } from "flagsmith-nodejs";

console.log(DefaultFlag);
```

```javascript
const Flagsmith = require("flagsmith-nodejs");

console.log(Flagsmith.DefaultFlag);
```

**Comment by matthewelwell** [comment]
Note that the current solution is to simply return an object in the same shape as the DefaultFlag class: 

```javascript
const handler = (featureName) => {
    return { value: "foo", enabled: true, isDefault: true }
}
```

**Comment by rolodato** [comment]
This is now solved since 4.0.0

```
> require('flagsmith-nodejs').DefaultFlag
[class DefaultFlag extends BaseFlag]
```
